js/main.js

```diff
@@ -0,0 +1,24 @@
+function justNumber (min, max) {
+  if (min < 0 || min >= max || max <= 0) {
+    return ('Не верный диапазон! Укажите другие значения.');
+  } else {
+    const value = (Math.random() * (max - min) + min);
+    const randomNumber = Math.round(value);
+    return randomNumber;
+  }
+}
+
+justNumber(2, 5);
+
+
+function numberWithPoint (min, max, afterPoint) {
+  if (min < 0 || min >= max || max <= 0) {
+    return ('Не верный диапазон! Укажите другие значения.');
+  } else {
+    const value = (Math.random() * (max - min) + min);
+    const randomNumber = value.toFixed(afterPoint);
+    return randomNumber;
```

Comment:
```suggestion
    return value.toFixed(afterPoint);
```

плюс есть нюанс в типизации, приходи на консультацию - расскажу